core/tokenRegistry/src/foreignAssets.ts

```diff
@@ -0,0 +1,132 @@
+// patch out annoying logs
+const info = console.info;
+console.info = function (x: any, ...rest: any) {
+  if (x !== 'secp256k1 unavailable, reverting to browser version') {
+    info(x, ...rest);
+  }
+};
+const warn = console.warn;
+console.warn = function (x: any, ...rest: any) {
+  if (
+    !x
+      .toString()
+      .startsWith(
+        'Error: Error: RPC Validation Error: The response returned from RPC server does not match the TypeScript definition. This is likely because the SDK version is not compatible with the RPC server.',
+      )
+  ) {
+    warn(x, ...rest);
+  }
+};
+
+import { ChainName, chains } from "@wormhole-foundation/sdk-base";
+import { ForeignAssetsCache, TokensConfig } from "./types";
+import { TokenId, Wormhole, toNative } from "@wormhole-foundation/connect-sdk";
+
+// TODO: Question: How do we handle if a user tries to perform an action for a chain/platform which isn't installed??
```

Comment:
I vote puke hard w/ msg about installing that package

Reply:
Def agree, broke it out into this issue https://github.com/wormhole-foundation/connect-sdk/issues/90